cmd/mtail/main.go

```diff
@@ -36,6 +36,7 @@ var logs seqStringFlag
 var (
 	port               = flag.String("port", "3903", "HTTP port to listen on.")
 	address            = flag.String("address", "", "Host or IP address on which to bind HTTP listener")
+	unixSocket         = flag.String("unix_socket", "", "UNIX Socket to listen on.")
```

Comment:
I know that jax0r has a period in `port`, but other lines don't have periods to end the sentence.

Reply:
I think we should go with no fullstop in the flag descriptions so please drop it.   I'll take care of `port`.